examples/vols/ex.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.1-dev
+
+services:
+- name: db
+  containers:
+  - image: mysql
+    mounts:
+    - volumeName: db
+      mountPath: /var/lib/mysql
+    - volumeName: iiit
+      mountPath: /var/lib/mysql
```

Comment:
This example is invalid!
```
ERROR: service "db": container#1: mount "iiit": mountPath "/var/lib/mysql": cannot have same mountPath as "db"
```

It would be nice to add example that is actually working and its meaningfully.
For example we could add another version of wordpress example with persistent storage for database